toolkit/tools/pkg/simplechroottool/simplechroottool.go

```diff
@@ -0,0 +1,146 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+// A common interface for creating a basic spec/rpm parsing environment inside a chroot.
+
+package simplechroottool
+
+import (
+	"path/filepath"
+	"runtime"
+
+	"github.com/microsoft/CBL-Mariner/toolkit/tools/internal/logger"
+	"github.com/microsoft/CBL-Mariner/toolkit/tools/internal/rpm"
+	"github.com/microsoft/CBL-Mariner/toolkit/tools/internal/safechroot"
+)
+
+const (
+	chrootSpecDirPath = "/SPECS"
+)
+
+// SimpleChrootTool is a tool for creating a basic spec/rpm parsing environment inside a chroot. When creating a new instance, the
+// following fields must be set:
+//   - BuildDirPath: The path to the directory where the chroot will be created and run from.
+//   - WorkerTarPath: The path to the tarball containing the chroot environment.
+// The following fields will be set by InitializeChroot():
+//   - chroot: The chroot environment we are working in, call Chroot.Run() to execute commands inside the chroot.
+//   - chrootSpecDir: The path to the SPECS directory inside the chroot "/SPECS."
+
+type SimpleChrootTool struct {
+	BuildDirPath  string
+	WorkerTarPath string
+	chroot        *safechroot.Chroot
+	chrootSpecDir string
+}
+
+// Chroot returns the chroot environment we are working in, call Chroot().Run() to execute commands inside the chroot. Call
+// InitializeChroot() before calling this function.
+func (s *SimpleChrootTool) Chroot() *safechroot.Chroot {
+	return s.chroot
+}
```

Comment:
Let's not expose "internals". Let's maybe go directly to `RunInChroot` or something.